src/rustfmt/main.rs

```diff
@@ -43,7 +43,27 @@ fn main() {
 
 /// Format Rust code
 #[derive(Debug, StructOpt, Clone)]
-#[structopt(name = "rustfmt", version = include_str!(concat!(env!("OUT_DIR"),"/version-info.txt")))]
+#[structopt(
+    name = "rustfmt",
+    version = include_str!(concat!(env!("OUT_DIR"),"/version-info.txt")),
+    about = r#"Format Rust code
+
+Rustfmt runs on a set of files or stdin. When invoked without any file arguments, rustfmt will
+read code from stdin.
+
+Please visit https://rust-lang.github.io/rustfmt to see all rustfmt configuration options.
+
+EXAMPLES
+
+    cat lib.rs | rustfmt
+        Feed the contents of "lib.rs" to rustfmt via stdin
+
+    rustfmt --emit=stdout lib.rs main.rs
+        Run rustfmt over "lib.rs" and "main.rs", writing to stdout (rather than in-place)
```

Comment:
IMO the examples here should preferably mimic the most common behavior and target the newest of users. I'd prefer to either drop `--emit=stdout` altogether, or if you'd like to leave it in there to highlight the emit flag, then I think we'd need to tweak the description to avoid giving anyone the impression that `emit` is required and/or that `--emit=files` is needed for the standard formatting mode.
Probably not literally this, but something that similarly conveys the message
```suggestion
    rustfmt --emit=stdout lib.rs main.rs
        Run rustfmt over "lib.rs" and "main.rs", writing to stdout (rather than in-place, which is the default without `--emit`)
```

Reply:
I added an example of the default emit above which I think covers your concern.

Reply:
That'll work, thanks for the update!